docker/swarm/localconfig_realslurm_oracle_zeke.sh

```diff
@@ -119,8 +119,8 @@ VCELL_OPT_SINGULARITY_IMAGE_EXTERNAL="/state/partition1/singularityImages/${VCEL
 VCELL_SMTP_HOSTNAME=vdsmtp.cam.uchc.edu
 VCELL_SMTP_PORT=25
 VCELL_SMTP_EMAILADDRESS=VCell_Support@uchc.edu
-VCELL_EXPORT_BASEURL=http://vcell.org/export/
-VCELL_EXPORTDIR_HOST=/Volumes/vcell/export/
+VCELL_EXPORT_BASEURL=http://localhost/
+VCELL_EXPORTDIR_HOST=/media/zeke/DiskDrive/Home/Work/CCAM/TempStorage/export
 VCELL_MAX_JOBS_PER_SCAN=100
```

Comment:
Don't forget to always update all 4 /docker/swarm/localconfig_xxx.sh config files as well as /docker/swarm/serverconfig-uch.sh whenever you add or modify environment variables.

Reply:
It seems as if most of the shell configuration files already had the enviromental vairables that would allow for the N5 export process to work. The biggest change was adding the env variable s3ProxyPortExternal to data-dev docker build file, and passing in the port to that environmental variable in the docker compose file.